### DISCUSSION TITLE
RingProgress, circle size.

### DISCUSSION THREAD

**Discussion by Kudienko** [mantinedev/mantine#6707] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.12.1

### What package has an issue?

@mantine/core

### What framework do you use?

Other, I will specify in the bug description

### In which browsers you can reproduce the issue?

All

### Describe the bug

I don't need the indentation, it doesn't work to make the actual size.
How can I get around this?
The only way to remove the indentation is by recalculating (size / 2) - (thickness / 2) or (75(size) /2) - (8(thickness ) /2) = r 33.5
How can this be solved from the outside?
There is just a layout, I am trying to use the mantine to the maximum, and according to the layout there should be no indentations.
![maket](https://github.com/user-attachments/assets/3e2118cb-d4fe-4beb-a732-c1139ba79ebb)
![image](https://github.com/user-attachments/assets/3d0955f2-5d6c-4c96-acbb-0a52d6d5645f)

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
To remove indentation from the component, use negative margins

**Reply by Kudienko** [reply]
The size of the circle will still be smaller

**Reply by Kudienko** [reply]
https://github.com/mantinedev/mantine/blob/master/packages/%40mantine/core/src/components/RingProgress/Curve/get-curve-props.ts
Can you explain why this particular formula for calculating the radius is used here? 
const radius = (size * 0.9 - thickness * 2) / 2;

**Reply by rtivital** [reply]
Not really, I took it from somewhere about 3 years ago. If RingProgress does not work as expected, try using DonutChart – https://mantine.dev/charts/donut-chart/
